pkg/operator/apiserver/audit/bindata/bindata.go

```diff
@@ -311,7 +311,54 @@ data:
       - group: "oauth.openshift.io"
         resources: ["oauthclients"]
     # catch-all rule to log all other requests with request and response payloads
-    - level: RequestResponse`)
+    - level: RequestResponse
+
+  userrequests.yaml: |
+    apiVersion: audit.k8s.io/v1beta1
+    kind: Policy
+    metadata:
+      name: UserRequests
+    # Don't generate audit events for all requests in RequestReceived stage.
+    omitStages:
+    - "RequestReceived"
+    rules:
+    # Logs request and response for oauthaccess tokens.
+    - level: RequestResponse
+      resources:
+      - group: "oauth.openshift.io"
+        resources: ["oauthaccesstokens"]
```

Comment:
we cannot log these if there are still old tokens in the system. @stlaz is it time to finally delete old tokens?

Reply:
I suppose it'd be the right time to do that

Reply:
Should I go to -level: None here? I think that would be better.